backend/views/views.py

```diff
@@ -105,22 +106,22 @@ def login():
     if form.validate_on_submit():
         user = db.session.query(User).filter(
             User.email == form.email.data).first()
-        if user and user.password == form.password.data:
+        if user.is_correct_password(form.password.data):
             session['user_id'] = user.id
             session['role_id'] = user.role_id
-            flash('Wellcome %s' % user.name)
+            flash('Welcome {}'.format(user.name))
```

Comment:
I'm not sure you need to change formating method here.

Reply:
fine :(